synfig-studio/src/gui/timeplotdata.cpp

```diff
@@ -136,7 +138,7 @@ TimePlotData::recompute_time_bounds()
 void
 TimePlotData::recompute_geometry_data()
 {
-	k = widget.get_width()/(upper - lower);
+	k = (widget.get_width()-EXTRA_TIMETRACK_SPACE*2) / (upper-lower);
```

Comment:
You're changing the behavior of a class method that is used by more than one class, ie. not only the one you're fixing. So be careful here: your fix can break other stuff.
TimePlotData is also used by Widget/Dock Sound and Graph, for example.

Therefore, instead of making such a fixed change by `#define` directive, use/create a class property and its accessor/mutator method.

Reply:
Well, that was the intention. Change the root class and let subclass follow. And define directive was the easier method because it's readable. BTW I haven't checked sound and graph source code so yes you're correct my change is breaking that two classes.

Looking at the timeplotdata...